app/helpers/patients_helper.rb

```diff
@@ -29,4 +29,11 @@ def fake_patient_params
                  birth_date: birth_date, phone: phone, email: email, street_line1: street_line1,
                  street_line2: street_line2, city: city, state: state, zip_code: zip_code } }
   end
+
+  def iis_patient_params
```

Comment:
Is there a reason for this to be here rather than in the test code?

Reply:
This is for a demo data button I made on the search form, copying the new patient form. I just moved it to the search helper but doesn't really belong in the test code. We also need to ensure this matches a patient in the IIS sandbox so I don't want to throw it in the controller.